File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/run_response.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class RunResponse:
    """运行响应"""
    id: str = None
    object: str = None
    status: str = None
    model: str = None
    instructions: str = None
    tools: List[Any] = None
    metadata: Dict[str, str] = None
    usage: Usage = None
    temperature: float = None
    created_at: int = None
    thread_id: str = None
    assistant_id: str = None
    required_action: RequiredAction = None
    last_error: LastError = None
    expires_at: int = None
    started_at: int = None
    cancelled_at: int = None
    failed_at: int = None
    completed_at: int = None
    tool_resources: Dict[str, Any] = None
    top_p: float = None
    max_prompt_tokens: int = None
    max_completion_tokens: int = None
    truncation_strategy: Any = None

File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/RunResponse.kt
package com.sdkwork.ai

data class RunResponse(
    val id: String? = null,
    val object: String? = null,
    val status: String? = null,
    val model: String? = null,
    val instructions: String? = null,
    val tools: List<Any>? = null,
    val metadata: Map<String, String>? = null,
    val usage: Usage? = null,
    val temperature: Double? = null,
    val createdAt: Int? = null,
    val threadId: String? = null,
    val assistantId: String? = null,
    val requiredAction: RequiredAction? = null,
    val lastError: LastError? = null,
    val expiresAt: Int? = null,
    val startedAt: Int? = null,
    val cancelledAt: Int? = null,
    val failedAt: Int? = null,
    val completedAt: Int? = null,
    val toolResources: Map<String, Any>? = null,
    val topP: Double? = null,
    val maxPromptTokens: Int? = null,
    val maxCompletionTokens: Int? = null,
    val truncationStrategy: Any? = null
)
